### PR TITLE
glusterfs:db_operations add indentation to json data

### PR DESCRIPTION
**Pull request by Madhu-1**
for DB export operation writes indented
output to JSON file.

Signed-off-by: Madhu <mrajanna@redhat.com>

### What does this PR achieve? Why do we need it?
with this heketi db export command will write the indented JSON output a file.

### CODE REVIEW

File: apps/glusterfs/db_operations.go [hunk 1]
```diff
@@ -226,8 +226,11 @@ func DbDump(jsonfile string, dbfile string) error {
 	if err != nil {
 		return fmt.Errorf("Could not construct dump from DB: %v", err.Error())
 	}
+	enc := json.NewEncoder(fp)
+	//set indentation for json data
```
Comment (phlogistonjohn) [inline apps/glusterfs/db_operations.go]: The comment here is a bit redundant as the next line is clearly setting the indent. I'd either reword the comment to provide more detail about why the indentation is being set OR just drop the comment.

Comment (obnoxxx) [reply apps/glusterfs/db_operations.go]: @phlogistonjohn, exactly! Avoid redundant comments!
@Madhu-1, please either remove the comment or explain what's going on.... :-)

**Comment by Madhu-1** [comment]
@obnoxxx  @phlogistonjohn  removed the comment.

**Comment by obnoxxx** [comment]
ok to test

**Pull request merged**